conda_manager/widgets/packages.py

```diff
@@ -379,6 +379,7 @@ def _run_conda_process(self, action, dic):
         """ """
         cp = self._conda_process
         prefix = self._prefix
+        print(action, dic)
```

Comment:
A real log line would be nice - any process executed in the background ought to be echoed to a terminal or logfile.